lib/src/wip/getErrorsMap.dart

```diff
@@ -5,15 +5,17 @@ import 'package:highlight/languages/java.dart';
 import 'package:highlight/languages/python.dart';
 import 'package:highlight/languages/scala.dart';
 
-import 'language_syntax/brackets_counting.dart';
```

Comment:
If you comment out the line below, it's better to also comment this out instead of deleting. That way we have less surprises when we uncomment the line below.